apps/api/src/app/shared/commands/commandValidationException.ts

```diff
@@ -0,0 +1,7 @@
+import { BadRequestException } from '@nestjs/common';
+
+export class CommandValidationException extends BadRequestException {
+  constructor(private mappedErrors: string[]) {
+    super(`Command validation failed: ${JSON.stringify(mappedErrors)}`);
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** I'm concerned that coercing the Error `message` to a string with JSON stringified errors will break existing API client usage of the error JSON. Could you please show an example before/after of the API response for this change?‏

Reply:
that was the original usage, just added a type to it so we can catch it, it was just throwing a bad request error

Reply:
The [original usage](https://github.com/novuhq/novu/pull/6648/files#diff-a55159e7e7743a752eb8fae910ad0fa0db836c21e2fb110c8e61b3c3e1885d16R23) returns the `mappedErrors` directly, it doesn't wrap them into a string. Am I missing something?